jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -189,12 +191,14 @@ def _find_line_items_by_invoice_and_distribution(cls, distribution_code_id, invo
     @classmethod
     def _get_invoices_for_disbursement(cls, partner):
         """Return invoices for disbursement."""
+        disbursement_date = datetime.today() - timedelta(days=current_app.config.get('DISBURSEMENT_DELAY_IN_DAYS'))
         invoices: List[InvoiceModel] = db.session.query(InvoiceModel) \
             .filter(InvoiceModel.invoice_status_code == InvoiceStatus.PAID.value) \
             .filter(
             InvoiceModel.payment_method_code.notin_([PaymentMethod.INTERNAL.value, PaymentMethod.DRAWDOWN.value])) \
             .filter((InvoiceModel.disbursement_status_code.is_(None)) |
                     (InvoiceModel.disbursement_status_code == DisbursementStatus.ERRORED.value)) \
+            .filter(~InvoiceModel.receipts.any(cast(ReceiptModel.receipt_date, Date) >= disbursement_date.date())) \
```

Comment:
Shouldn't this be **<=** ? 
For e.g, if a PAD payment receipt is created for Monday, then when the job runs on Friday it should pick all invoices which are PAID and have receipt date less than or equal to monday right ? 🤔

Reply:
hm..I applied the ~ , the negation..without negation , we can do the logic <=.. so the logic checks now ;if there is no receipt after the disbursement date ...
The only reasoning for that is incase if there are multiple receipts and one of them is after disbursement date , it shudnt disbursed...

Reply:
<= is good enough if no multiple receipts scenario is in place..

Reply:
didn't notice the **~**.